### PR TITLE
feat: add audit logs

### PR DESCRIPTION
**Pull request by thiesen**

### CODE REVIEW

File: README.tfdoc.hcl [hunk 1]
```diff
@@ -210,6 +210,68 @@ section {
...
+          description    = <<-END
+            List of audit logs settings to be enabled.
+          END
+          readme_example = <<-END
+            audit_configs = [{
+              service = "allServices"
+              configs = [{
+                log_type = "DATA_WRITE"
+              }]
+            }]
```
Comment (soerenmartius) [inline README.tfdoc.hcl:225]:
```suggestion
            # Enable full audit log coverage for all services
            audit_configs = [
                {
                    service = "allServices"
                    configs = [
                        {
                            log_type = "DATA_READ"
                        },
                        {
                            log_type = "DATA_WRITE"
                        },
                        {
                            log_type = "ADMIN_READ"
                        },
                        {
                            log_type = "ADMIN_WRITE"
                        },
                    ]
                }
            ]
```

Comment (soerenmartius) [reply README.tfdoc.hcl:225]: let's add a more comprehensive example for this case

**Review by soerenmartius — CHANGES_REQUESTED** [review]

File: README.tfdoc.hcl [hunk 2]
```diff
@@ -210,6 +210,68 @@ section {
...
+            attribute "log_type" {
+              required    = true
+              type        = string
+              description = <<-END
+                Permission type for which logging is to be configured.
+                Must be one of `DATA_READ`, `DATA_WRITE`, or `ADMIN_READ`.
+              END
+            }
+
+            attribute "exempted_users" {
```
Comment (1fawadshah) [inline README.tfdoc.hcl:266]: It should be "exempted_members"

**Review by soerenmartius — CHANGES_REQUESTED** [review]
Could you add the changes to the `CHANGELOG.md` also please? thanks!

File: README.tfdoc.hcl [hunk 3]
```diff
@@ -210,6 +210,79 @@ section {
             }
           }
         }
+
+        variable "audit_configs" {
+          type = object(audit_log)
```
Comment (mariux) [inline README.tfdoc.hcl:215]:
```suggestion
          type = object(audit_config)
```

**Review by mariux — CHANGES_REQUESTED** [review]

**Review by kassianh — APPROVED** [review]
lgtm!

**Comment by thiesen** [comment]
hey @soerenmartius - I just pushed the changelog changes - please advise if the text is good enough. 

sorry for the delay - I somehow missed the request for updating it; thanks @kassianh for pointing it out

**Review by soerenmartius — DISMISSED** [review]
lgtm! @mariux are we good to go here?

**Review by mariux — APPROVED** [review]
lgtm

**Pull request merged**